src/ralph/dns/dnsaas.py

```diff
@@ -242,29 +255,36 @@ def _post(self, url, data):
         Returns:
             tuple (response status code, dict data)
         """
+        self._verify_oauth_token_validity()
         response = self._send_request_to_dnsaas('POST', url, json_data=data)
         return response.status_code, self._response2result(response)
 
-    def _delete(self, url):
+    @renew_token_when_unauthorized
+    def _delete(self, url: str) -> [int, Optional[dict]]:
+        self._verify_oauth_token_validity()
         response = self._send_request_to_dnsaas('DELETE', url)
 
         return response.status_code, self._response2result(response)
 
-    def _get(self, url):
+    @renew_token_when_unauthorized
+    def _get(self, url: str) -> [int, Optional[dict]]:
+        self._verify_oauth_token_validity()
         response = self._send_request_to_dnsaas('GET', url)
 
         return response.status_code, self._response2result(response)
 
-    def _patch(self, url, data):
+    @renew_token_when_unauthorized
+    def _patch(self, url: str, data: dict) -> [int, Optional[dict]]:
+        self._verify_oauth_token_validity()
```

Comment:
gdyby `_verify_oauth_token_validity` bylo w dekoratorze to nie trzeba byloby pamietac o wolaniu tej metody co chwile